app/src/main/java/com/asfoundation/wallet/wallet_reward/RewardFragment.kt

```diff
@@ -74,6 +79,7 @@ class RewardFragment : BasePageViewFragment(), SingleStateFragment<RewardState,
     super.onResume()
     viewModel.fetchPromotions()
     viewModel.fetchGamificationStats()
+    navBarViewModel.clickedItem.value = Destinations.REWARDS.ordinal
```

Comment:
It seems like this is not working when coming back from another activity. for example: Rewards screen > settings > back press > home will be selected

It can be fixed by adding a small delay (although not a good solution):
```suggestion
    lifecycleScope.launch { 
      delay(10L)
      navBarViewModel.clickedItem.value = Destinations.REWARDS.ordinal 
    }
```